java-checks/src/main/java/org/sonar/java/checks/regex/CanonEqFlagInRegexCheck.java

```diff
@@ -93,7 +94,9 @@ private static class CharacterVisitor extends RegexBaseVisitor {
 
     @Override
     public void visitSequence(SequenceTree tree) {
-      subjectToNormalization.addAll(getGraphemeInList(tree.getItems()));
+      subjectToNormalization.addAll(getGraphemeInList(tree.getItems()).stream()
```

Comment:
what about not collecting into a list but directly add in `subjectToNormalization`?
```java
getGraphemeInList(tree.getItems())
  .stream()
  .map(RegexIssueLocation::fromCommonsRegexIssueLocation)
  .forEach(subjectToNormalization::add)
```

Reply:
Indeed, way better!